app/controllers/merchants_items_controller.rb

```diff
@@ -0,0 +1,32 @@
+class MerchantsItemsController < ApplicationController
```

Comment:
New controller! This fixes the permissions issues we found when adding items

Reply:
![](https://media0.giphy.com/media/XfD8VJDUurgMjNEP72/giphy.gif)

Reply:
What Michael and that lady said